rust-runtime/aws-smithy-runtime-api/src/client/interceptors/context.rs

```diff
@@ -222,7 +222,6 @@ impl<I, O, E> InterceptorContext<I, O, E> {
     }
 
     /// Advance to the Serialization phase.
-    #[doc(hidden)]
```

Comment:
All these interceptor context ones should say they're not intended to be used

Reply:
Good call out. Addressed in [55f1c75](https://github.com/smithy-lang/smithy-rs/pull/3226/commits/55f1c75559c80281eef9cac17db3f54f253d7cc7).